migrations/dev-20210823-change-event-members-from-backers-to-attendees.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(
+      `UPDATE "Members" SET "role" = 'ATTENDEE' FROM "Tiers" t, "Collectives" c WHERE c.type = 'EVENT' AND role != 'ATTENDEE' AND t.type = 'TICKET' AND "TierId" = t.id AND t."CollectiveId" = c.id`,
```

Comment:
Some formatting could make this more readable, but the query itself seems good :+1:

Reply:
Thanks. Corrected it. 👍🏽